server/posts.tsx

```
@@ -0,0 +1,138 @@
+        await prisma.comment.deleteMany({
+            where: { postId: id },
+        });
+
+        // Then delete the post
+        const deletedPost = await prisma.post.delete({
+            where: { id: id },
+        });
+
+        return deletedPost;
+    }
+    catch (error: any) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)